packages/component/src/Utils/parseDocumentFragmentFromString.ts

```diff
@@ -0,0 +1,8 @@
+export default function parseDocumentFragmentFromString(html: string): DocumentFragment {
+  const parser = new DOMParser();
+  const fragment = new DocumentFragment();
+
+  fragment.append(...parser.parseFromString(html, 'text/html').body.childNodes);
+
+  return fragment;
```

Comment:
```suggestion
  const parser = new DOMParser();
  const parsedDocument = parser.parseFromString(html, 'text/html'
  const fragment = parsedDocument.createDocumentFragment();

  fragment.append(parsedDocument.body.childNodes);

  return fragment;
```
If we want to save the `parsedDocument` ownership of any element added to the fragment.

Reply:
This works much better... my previous code will run `onload` script. 🙈

Good that we have test. 😉